libs/velo-external-db-core/src/web/domain-to-spi-error-translator.ts

```diff
@@ -0,0 +1,25 @@
+import { errors as domainErrors} from '@wix-velo/velo-external-db-commons' 
```

Comment:
IMO we create unit tests on that function (even though it is pretty straightforward), just as we do for other translators.